package.json

```diff
@@ -35,7 +35,7 @@
     "chai-as-promised": "^6.0.0",
     "mocha": "^3.2.0",
     "rimraf": "^2.5.4",
-    "ts-node": "^1.7.2",
+    "ts-node": "^2.0.0",
```

Comment:
Zero idea.

From time to time, I check versions of packages and upgrade things, then do tests to see if it broke anything.

It didn't break anything!